### DISCUSSION TITLE
解锁码错误

### DISCUSSION THREAD

**Discussion by ZerkyLiu** [Meloong-Git/PCL#358] [提问] [answered]
我之前在旧电脑上使用了解锁码，现在换了个新电脑，输入解锁码时显示解锁码有误。我也不知道怎么办，难道解锁码是一次性的？😥

**Comment by WForst-Breeze** [comment, marked as answer]
你是指赞助了23.33档的解锁码嘛？每个解锁码只能在**有与之对应的识别码的设备上使用**，说人话就是差不多是一次性的。如果你更换了设备，可以前往爱发电把你的新设备的识别码（更多-关于与鸣谢-赞助者-复制识别码）私信龙猫获取新的解锁码😉

**Reply by WForst-Breeze** [reply]
补充：如果是极客蓝的解锁码就没有过期这一说了

**Comment by ZerkyLiu** [comment]
哦，知道了，谢谢
